cli/src/pcluster/utils.py

```diff
@@ -32,11 +32,29 @@
 from yaml.resolver import BaseResolver
 
 from pcluster.aws.common import get_region
-from pcluster.constants import SUPPORTED_OSES_FOR_ARCHITECTURE, SUPPORTED_OSES_FOR_SCHEDULER
+from pcluster.constants import SUPPORTED_OSES_FOR_ARCHITECTURE, SUPPORTED_OSES_FOR_SCHEDULER, UNSUPPORTED_FEATURES_MAP
 
 LOGGER = logging.getLogger(__name__)
 
 
+def is_feature_supported(feature: str, region: str = None):
```

Comment:
If we create the generic `FeatureRegionValidator` this code can be moved there, removing this utils function.

Reply:
In we agree on having the validator that way, I'll do that.

Reply:
Let's do it!